File: Python/day1-1.py
with open("Recursos/day1-input.txt") as file:
    expenses = [int(i) for i in file.readlines()]

for i in range(len(expenses)):
    for j in range(i, len(expenses)):
        v1 = expenses[i]
        v2 = expenses[j]
        if v1 + v2 == 2020:
            print(
                f"{v1} + {v2} = {v1 + v2 }")
            print(
                f"{v1} * {v2} = {v1 * v2 }")
            break

File: Julia/day1-1.jl
using DelimitedFiles

expenses = readdlm("Recursos/day1-input.txt", Int)

for i = 1:length(expenses)
    for j = i:length(expenses)
        if expenses[i] + expenses[j] == 2020
            println("$(expenses[i]) + $(expenses[j]) == $(expenses[i] + expenses[j])")
            println("$(expenses[i]) * $(expenses[j]) == $(expenses[i] * expenses[j])")
            break
        end
    end
end
